utilities/general-helpers.metta

```diff
@@ -207,3 +207,153 @@
           )
           (cons-atom ($x $y) $rest)))
 )
+
+;; Finds length of a tuple
+
+(= (len $expr)
+    (if (== $expr ())
+        0
+        (+ 1 (len (cdr-atom $expr)))
```

Comment:
Just curious about this one, will it have effect on it's performance if you use the `let` binding here instead of calling the recursion like you did?